cloudconfig/podcfg/image.go

```diff
@@ -61,30 +61,19 @@ func IsCharmBaseImage(imagePath string) bool {
 // GetJujuOCIImagePath returns the jujud oci image path.
 func GetJujuOCIImagePath(controllerCfg controller.Config, ver version.Number) (string, error) {
 	// First check the deprecated "caas-operator-image-path" config.
-	imagePath, err := RebuildOldOperatorImagePath(
-		controllerCfg.CAASOperatorImagePath().Repository, ver,
```

Comment:
This is showing functionally a change in behaviour, BUT, I think this might be ok, since CAASOperatorImagePath was always an OCI reference. CAASImageRepo is either a partial OCI reference or JSON describing a repo with credentials etc etc...